terraform/README.md

```diff
@@ -1,114 +1,232 @@
-# Setting up an automatic review job
+# GKE Policy Automation deployment
 
-## Introduction
+A Terraform code for deploying GKE Policy Automation as an automatic serverless solution
+on Google Cloud Platform.
 
-To receive regular reports on the status of your GKE cluster, you can deploy the GKE Policy Review t
-ool as a serverless CRON job on GCP.
+The solution leverages the below GCP components:
 
-[Cloud Run Jobs](https://cloud.google.com/run/docs/triggering/using-scheduler) is the perfect tool
-to run such a job. Cloud Run Jobs can be triggered by a Cloud Scheduler on any CRON schedule and
-only consume resources during the time your job is actually actively running.
+* [Cloud Scheduler](https://cloud.google.com/scheduler)
+  to trigger execution of a GKE Policy Automation tool in a periodic manner
+* [Cloud Run Jobs](https://cloud.google.com/run/docs/create-jobs)
+  to run containerized GKE Policy Automation tool
+* [Artifact Registry](https://cloud.google.com/artifact-registry)
+  to store GKE Policy Automation tool container image locally
+* [Cloud Asset Inventory](https://cloud.google.com/asset-inventory/docs/overview)
+  to discover GKE clusters in GCP organization, selected folders or projects
+* Optionally, [Cloud Storage](https://cloud.google.com/storage),
+  [Cloud Pub/Sub](https://cloud.google.com/pubsub), or
+  [Security Command Center](https://cloud.google.com/security-command-center)
+  as destinations for cluster evaluation results
 
-Unfortunately, this service is currently only available in one region and not supported by Terraform
-yet. While most of this setup can easily be deployed by running this Terraform script, Cloud Run
-Jobs have to be deployed manually with the gcloud CLI until the Terraform provider is available.
+![GKE Policy Automation infrastructure](../assets/gke-policy-automation-infra.png)
 
-## Before you begin
+---
 
-Before running the script and the additional gcloud commands, please set the following environment
-variables:
+## Prerequisites
 
-```bash
-export TF_VAR_job_name="gke-policy-automation-job"
-```
+* Terraform installed
+* `gcloud` command
+* Exiting project for GKE Policy Automation resources
+* IAM permissions to create resources in the GKE Policy Automation project
+* IAM permissions to create new IAM role bindings on projects, folders or organization levels
+  *(depending on desired cluster discovery or outputs configuration)*
 
-```bash
-export TF_VAR_project_id="YOUR GCP PROJECT ID"
-```
+## Running
 
-```bash
-export TF_VAR_region="YOUR GCP REGION, e.g. europe-west1"
-```
+Provision infrastructure with Terraform:
 
-Adjust the `config.yaml` file according to your preferences. The provided example will use cluster discovery
-on a GCP project, Cloud Storage output for reports and the default set of GKE policies.
+1. Set Terraform configuration variables *(check [examples](#example-configurations)
```

Comment:
maybe we can put here some minimal configuration (eg. sample .tfvars file + command tf apply -var-file sample.tfvars)? (just to avoid hopping through documentation while setting this up? 
or maybe we can generate here inputs documentation with tfdoc?

Reply:
Added an example.